docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -95,13 +98,104 @@ You must reload the page after disabling extensions or display issues may occur.
 
 :::
 
-## Updating and Upgrading Extensions
+## Developing Extensions
 
-1. Click **☰ > Extensions** under **Configuration**.
-1. Select the **Updates** tab. 
-1. Click **Update**.
+To learn how to develop your own extensions, refer to the official [Getting Started](https://rancher.github.io/dashboard/extensions/extensions-getting-started) guide.
 
-If there is a new version of the extension, there will also be an **Update** button visible on the associated card for the extension in the **Available** tab.
+## Working with Extensions in an Air-gapped Environment
+
+If you intend to work with extensions in an air-gapped environment, you must perform some extra steps before you can complete certain tasks.
+
+### Accessing Rancher UI Extensions in an Air-Gapped Environment
+
+Rancher provides some extensions, such as Kubewarden and Elemental, through the `ui-plugin-catalog` container image at https://hub.docker.com/r/rancher/ui-plugin-catalog/tags. If you're trying to install these extensions in an air-gapped environment, you must make the `ui-plugin-catalog` accessible.
+
+1. Mirror the `ui-plugin-catalog` image to a private registry:
+```bash
+export REGISTRY_ENDPOINT="my-private-registry.com"
+docker pull rancher/ui-plugin-catalog:1.0.0
+docker tag rancher/ui-plugin-catalog:1.0.0 $REGISTRY_ENDPOINT/rancher/ui-plugin-catalog:1.0.0
+docker push $REGISTRY_ENDPOINT/rancher/ui-plugin-catalog:1.0.0
+```
```

Comment:
```suggestion

  ```bash
  export REGISTRY_ENDPOINT="my-private-registry.com"
  docker pull rancher/ui-plugin-catalog:1.0.0
  docker tag rancher/ui-plugin-catalog:1.0.0 $REGISTRY_ENDPOINT/rancher/ui-plugin-catalog:1.0.0
  docker push $REGISTRY_ENDPOINT/rancher/ui-plugin-catalog:1.0.0
  ```

```

Reply:
Updating formatting for this list of instructions:

<img width="769" alt="Screenshot 2024-04-05 at 10 10 32 AM" src="https://github.com/rancher/rancher-docs/assets/83089978/4352ad27-6697-4dc0-9499-a299bad48830">